### ISSUE TITLE
Next 13.1 compat

### ISSUE THREAD

**Issue by MoOx** [nissy-dev/swc-plugin-react-native-web#34]
I am creating a project with RNW+Next.js and I am having an issue when trying to install your SWC plugin with npm.

```console
$ nis @nissy-dev/swc-plugin-react-native-web
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: nws@0.1.0
npm ERR! Found: next@13.1.1
npm ERR! node_modules/next
npm ERR!   next@"13.1.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer next@"~13.0.0" from @nissy-dev/swc-plugin-react-native-web@0.3.0
npm ERR! node_modules/@nissy-dev/swc-plugin-react-native-web
npm ERR!   @nissy-dev/swc-plugin-react-native-web@"*" from the root project
```

I guess playing with SWC & next is touchy so you prefer to use `~` for deps instead of `^` ?

**Comment by nissy-dev** [comment]
@MoOx 
The reason I use `~` instead of `^` is that SWC plugins are often broken by updating Next.js npm package even if it's a minor version upgrade.  

Please confirm https://github.com/nissy-dev/swc-plugin-react-native-web#compatibility. And then, if you are using Next.js, I think this plugin is not needed in many cases. Please try [package-aliasing](https://necolas.github.io/react-native-web/docs/setup/#package-aliasing) and read necolas/react-native-web#2217

**Comment by nissy-dev** [comment]
I released the new package for Next 13.1.
